app/policies/post_policy.rb

```diff
@@ -1,7 +1,11 @@
 class PostPolicy < ApplicationPolicy
 	class Scope < Scope
 		def resolve
-			scope
+			if user.admin?
+				scope.all
+			else
+				scope.where(published: true)
+			end
```

Comment:
This policy is not being used yet, but will keep and experiment for a bit before deciding to remove.